README.md

```diff
@@ -25,7 +25,8 @@ For example, a hook which targest python will be called `python-...`.
 
 [generated]: # (generated)
 - **`python-check-blanket-noqa`**: Enforce that `noqa` annotations always occur with specific codes
-- **`python-check-mock-methods`**: Prevent a common mistake of `assert mck.not_called()` or `assert mck.called_once_with(...)`
+- **`python-check-mock-methods`**: Prevent common mistakes of `assert mck.not_called()`, `assert mck.called_once_with(...)` and and `mck.assert_called`.
+
```

Comment:
the `>` is causing a weird linebreak here

Reply:
Missing a `strip()` in the README generator?

Reply:
I'd rather not, the change as written adds a newline to the end of the `description`